tools/install-unprivileged.sh

```diff
@@ -179,11 +219,11 @@ latesturl()
 	fi
 }
 
-APPTAINERURL="https://kojipkgs.fedoraproject.org/packages/apptainer/"
 LOCALAPPTAINER=false
-if [ -z "$VERSION" ]; then
+#if test -z "$VERSION" || ! test -z $OBSURL ; then
+if test -z "$VERSION" || [[ "$KOJI" == "false" ]]; then
```

Comment:
Let's not leave this comment, and don't make the style inconsistent by changing `[ -z` to `test`.